pegasus/sites.v3/code.org/public/teacher-dashboard/index.haml

```diff
@@ -102,7 +102,10 @@ angular: teacherDashboard
       //  'remove': {method:'DELETE'},
       //  'delete': {method:'DELETE'}
          update: {method:'POST', url: 'v2/sections/:id/update'},
+         // TODO: This isn't very restful
+         allStudents: {method:'GET', url:'v2/sections/:id/students', isArray: true},
          addStudents: {method:'POST', url:'/v2/sections/:id/students', isArray: true},
+         moveStudents: {method:'POST', url:'/sections/:new_section_code/transfers'},
```

Comment:
@laurelfan I think this is the code that hits the wrong domain. Maybe I need to make the API call in a different way? Or maybe I'm just completely missing something, which is highly possible :grimacing: I'll also have some time this weekend to look this over again too.